device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,484 @@
+            },
+            {
+                "name": "thermal 29",
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 30",
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 31",
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 32",
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 33",
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 34",
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 35",
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 36",
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 37",
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 38",
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 39",
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            },
+            {
+                "name": "thermal 40",
+                "high_threshold": 100,
+                "high_critical_threshold": 100,
+                "low_threshold": 10,
+                "low_critical_threshold": 0
+            }
+        ],
+        "sfps": [
+            {
+                "name": "sfp 1"
```

Comment:
we don't have a name for sfps?

Reply:
The API has a `get_name()` method (in the `DeviceBase` class). This should be implemented and provide a name (even if it's `sfp1`).

Reply:
Changed the names of sfps to to sfp1, sfp2 ..., will need an input from Mellanox to confirm

Reply:
@keboliu Is Mellanox planning to implement the returning the name of Transceivers ? As @jleveque mentioned 
The API has a get_name() method (in the DeviceBase class) for all devices